### ISSUE TITLE
[Important] Deprecation Notice

### ISSUE THREAD

**Issue by timroes** [timroes/EnhancedListView#58]
I added a deprecation notice in favor of the new RecyclerView coming soon in the support library.

Please make sure to read the updated README file.

**Comment by kenyee** [comment]
does recyclerview support swipe to dismiss?  I didn't see anything that mentioned it...

**Comment by timroes** [comment]
No. Recycler view doesn't support swipe to dismiss on its own. But any library that works on ListView should be ported to the new RecyclerView as soon as possible (imho) to (a) benefit from all the other features Recycler View offers and (b) not relying on "old" techniques. E.g. recycler view has a very different support for animations, that should be leveraged for that kind of library.

**Comment by Binghammer** [comment]
Do you plan to implement a swipe-to-dismiss to the RecyclerView? I'm finding that setting an onClickListener will cancel out the dismiss listener. Very frustrating.

**Comment by joan38** [comment]
I don't understand why its not built in Recyclerview. It is implemented in iOS for a long time now!

**Comment by timroes** [comment]
@joan38: good question and I am not sure if it might not come in the future. Thats one of the advantages of having the stuff in the support library, they can update it some time to e.g. add new layout managers or implement a feature like that.

@Binghammer unfortunately I don't see that I will have the time to completely rewrite this. I hope someone else does so and perhaps there is some useful code in this library that can be used.

**Comment by LucioC** [comment]
@timroes @Binghammer I forked this project and started to use the RecyclerView https://github.com/LucioC/EnhancedList . I already have a functional version, but needs some review. I appreciate feedbacks. I'm still using the old animations, maybe this need to be improved.

**Comment by joan38** [comment]
This project might help:
https://github.com/rahulrj/Swipe_RecyclerView

**Comment by Binghammer** [comment]
Few others here: 
https://github.com/Malinskiy/SuperRecyclerView
https://github.com/ismoli/DynamicRecyclerView
https://github.com/emileb/DragSortRecycler
https://github.com/vinc3m1/DragSortAdapter
